docs/operating-scylla/procedures/cluster-management/add-node-to-cluster.rst

```diff
@@ -2,14 +2,14 @@
 Adding a New Node Into an Existing Scylla Cluster (Out Scale)
 ==============================================================
 
-When you add a new node, another node in the cluster streams data to the new node. This operation is called bootstrapping and may
+When you add a new node, other nodes in the cluster stream data to the new node. This operation is called bootstrapping and may
 be time-consuming, depending on the data size and network bandwidth. If using a :ref:`multi-availability-zone <faq-best-scenario-node-multi-availability-zone>`, make sure they are balanced.
 
 
 Prerequisites
 -------------
 
-Before adding the new node, check the node's status in the cluster using :doc:`nodetool status </operating-scylla/nodetool-commands/status>` command.
+#. Before adding the new node, check the node's status in the cluster using :doc:`nodetool status </operating-scylla/nodetool-commands/status>` command.
 You cannot add new nodes to the cluster if any nodes are down.
```

Comment:
```suggestion
#. Before adding the new node, check the node's status in the cluster using :doc:`nodetool status </operating-scylla/nodetool-commands/status>` command. You cannot add new nodes to the cluster if any nodes are down.
```

Reply:
BTW, the above statement is not correct in a general case.
You *can* add nodes to the cluster if some nodes are DOWN as long as those node are not the ones that are going to loose tokens due to the topology change.

For example, if all replicated KSes use NetworkingTopologyStrategy and RF in the current DC is 3 and there are 3 racks in the cluster in the current DC, then adding a new node should not be blocked by any number of DOWN nodes in racks different from the one a node is being added to.

Reply:
As a rule, a new node should not be added if an existing node is down, and this page documents that simple rule.
I understand there are cases where this rule will not hold - the question is how to document it (however we do it, let's take it out outside of this PR).
@vladzcloudius  Could we start by adding a new post to the forum? I'd add a question, and you'd provide the answer. Would that be OK?